sdk/netapp/Azure.ResourceManager.NetApp/src/Custom/Models/VaultList.cs

```diff
@@ -1,11 +1,11 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Since nothing public changed in this file, we could just remove this file from this directory and regenerate to take its generated version.